apps/storybook/.storybook/preview.js

```diff
@@ -55,6 +55,7 @@ export const decorators = [
     } = context;
 
     React.useEffect(() => {
+      document.body.classList.toggle('iui-root', true);
```

Comment:
this resulted in the Icon test image changing ever so slightly. i noticed that the old size was 16x16 whereas now it's 15.99x15.99 for some reason 😄

Reply:
there is another "random" test image change. this one can be explained by a change in font size.
![](https://user-images.githubusercontent.com/9084735/216459607-e60ee80a-5a11-45fe-96f0-222f5e824d96.png)

see main branch: https://itwin.github.io/iTwinUI/react/iframe.html?args=&id=utilities-flex--basic
vs this branch: https://itwin.github.io/iTwinUI/1031/react/iframe.html?args=&id=utilities-flex--basic